server/api/challonge.js

```diff
@@ -0,0 +1,65 @@
+const ChallongeService = require('../services/ChallongeService.js');
+const logger = require('../log.js');
+const passport = require('passport');
+const { wrapAsync } = require('../util.js');
+
+let challongeService = new ChallongeService();
+
+module.exports.init = function(server) {
+    server.get('/api/challonge/tournaments/', passport.authenticate('jwt', { session: false }), wrapAsync(async function(req, res) {
+        challongeService.getTournamentsForUser({ ...req.user })
```

Comment:
Here you're using wrapAsync but still using the .then/.catch syntax.
You can change it to a try/catch and use await